scene/animation/tween.cpp

```diff
@@ -228,6 +228,14 @@ Ref<Tween> Tween::set_loops(int p_loops) {
 	return this;
 }
 
+int Tween::get_loops_left() const {
+	if (loops == 0) {
```

Comment:
```suggestion
	if (loops <= 0) {
```
Values below 0 are accepted in `set_loops()`, but they have the same effect as 0.